src/main/java/ui/RepositorySelector.java

```diff
@@ -49,6 +52,17 @@ private void setupComboBox() {
                 onValueChangeCallback.accept(repoId);
             }
         });
+        
+        // remove focus from the repo selector
+        comboBox.addEventFilter(KeyEvent.KEY_RELEASED, event -> {
+            event.consume();
+            if (event.getCode() == KeyboardShortcuts.REMOVE_FOCUS) {
+                if (ui.isTestMode()) {
```

Comment:
hmm the thing is we are _removing_ focus. Plus to keep consistent with other event related testing, shouldn't we stick to this approach? Or we switch event based testing to your suggestion